test_ai_aggies.py

```diff
@@ -1,7 +1,6 @@
-"""Tests for Team AI Aggies"""
+""""Tests for Team AI Aggies"""
 import unittest
-from pii_scan import show_aggie_pride,  anonymize_text
-
+from pii_scan import show_aggie_pride, anonymize_text
```

Comment:
It looks like a blank line was deleted here?  It is always best to preserve the original formatting otherwise you will potentially generate merge conflicts.